### PR TITLE
feat:add colorSheme, add colors

### PR DESCRIPTION
**Pull request by HarryPotter-4000**
### Description

Please provide a brief description of the changes you have made in your PR.

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [x] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Mantine components/constants/hooks have been used wherever possible

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change

Add screenshots of the changed components here.

#### Screenshots/Video after change
image
image

Add screenshots with the changes applied here.

### CODE REVIEW

File: src/components/ApplicationShell.jsx [hunk 1]
```diff
@@ -12,37 +12,42 @@ import {
   UnstyledButton,
   useMantineTheme,
 } from "@mantine/core";
+//import { useClickOutside } from "@mantine/hooks";
```
Comment (LizavetaY) [inline src/components/ApplicationShell.jsx]: Delete comments, please :)

File: src/components/ApplicationShell.jsx [hunk 2]
```diff
@@ -12,37 +12,42 @@ import {
...
+  const { colors } = useMantineTheme();
   const theme = useMantineTheme();
+
   const [isOpened, setIsOpened] = useState(false);
   const [, setIsUserMenuOpened] = useState(false);
 
   const toggleMenu = () => setIsUserMenuOpened((bool) => !bool);
+
   const toggleBurger = () => setIsOpened((bool) => !bool);
+  //useClickOutside(() => setIsUserMenuOpened(false), ["mouseup", "touchend"]);
```
Comment (LizavetaY) [inline src/components/ApplicationShell.jsx]: The comment

File: src/components/ApplicationShell.jsx [hunk 3]
```diff
@@ -12,37 +12,42 @@ import {
...
   return (
     <AppShell
-      styles={{
-        main: {
-          background:
-            theme.colorScheme === "dark"
-              ? theme.colors.dark[8]
-              : theme.colors.gray[0],
-        },
+      //bg={colors.bg0}
```
Comment (LizavetaY) [inline src/components/ApplicationShell.jsx]: The comment

File: src/components/ApplicationShell.jsx [hunk 4]
```diff
@@ -55,13 +60,14 @@ export const ApplicationShell = () => {
               <Burger
                 opened={isOpened}
                 onClick={toggleBurger}
-                size="sm"
-                color={theme.colors.gray[6]}
+                size="18px"
```
Comment (lazzzaroni) [inline src/components/ApplicationShell.jsx]: Why did you change value?
"sm" here produce the same result as 18px ("sm" == 1.125rem == 18px).

I suggest not to use hardcoded values, even if result will be the same.

Comment (HarryPotter-4000) [reply src/components/ApplicationShell.jsx]: of course you are right

Comment (lazzzaroni) [reply src/components/ApplicationShell.jsx]: It's not resolved yet 🙂

**Review by lazzzaroni — APPROVED** [review]

**Review by LizavetaY — APPROVED** [review]
